lib/_tls_common.js

```diff
@@ -240,7 +240,7 @@ exports.translatePeerCertificate = function translatePeerCertificate(c) {
     c.infoAccess = Object.create(null);
 
     // XXX: More key validation?
-    info.replace(/([^\n:]*):([^\n]*)(?:\n|$)/g, function(all, key, val) {
+    info.replace(/([^\n:]*):([^\n]*)(?:\n|$)/g, function replacerCallback(all, key, val) {
```

Comment:
Switch to an anonymous function instead of naming it? The function name is not particularly useful.

Reply:
@apapirovski I think the point is to name anonymous functions so that stack traces and heap-debugging are more understandable.

Reply:
@apapirovski any update on this?

Reply:
I implemented the suggestions and fixed the conflicts. Happy to go ahead if all the checks are green?